modules/pubwiseBidAdapter.js

```diff
@@ -532,9 +532,8 @@ function _cleanSlotName(slotName) {
 
 function _initConf(refererInfo) {
   return {
-    // TODO: do the fallbacks make sense here?
-    pageURL: refererInfo?.page || window.location.href,
-    refURL: refererInfo?.ref || window.document.referrer
+    pageURL: (refererInfo && refererInfo.referer) ? refererInfo.referer : window.location.href,
+    refURL: window.document.referrer
```

Comment:
Why aren't you using refererInfo. Ref here?

Reply:
@patmmccann not sure, I'll check, but looks like we should be

Reply:
refererInfo.ref and refererInfo.page have a bunch of logic attempting to determine them as of Prebid 7 merge

Reply:
This has been adjusted in the next update.